CHANGELOG.md

```diff
@@ -12,7 +13,6 @@
 * [Fix] Fix error when using SQL Server with pyodbc that caused queries to fail due to multiple open result sets
 * [Fix] Improves performance when converting DuckDB results to `pandas.DataFrame`
 * [Fix] Fixes a bug when converting a CTE stored with `--save` into a `pandas.DataFrame` via `.DataFrame()`
-* [Doc] Add Redshift tutorial
```

Comment:
Please revert this deleted entry

Reply:
[fixed](https://github.com/ploomber/jupysql/pull/789/files#diff-06572a96a58dc510037d5efa622f9bec8519bc1beab13c9f251e97e657a9d4edR2-R8)